static/app/views/alerts/rules/metric/eapField.spec.tsx

```diff
@@ -111,4 +99,37 @@ describe('EAPField', () => {
     expect(screen.getByText('count')).toBeInTheDocument();
     expect(screen.getByText('spans')).toBeInTheDocument();
   });
+
+  it('defaults count_unique argument to user', async function () {
```

Comment:
The test title says it defaults to `user` but it seems like the default in the test is `span.op`, should we just change it to `span.op`?